docs/releases/2.2.3.txt

```diff
@@ -0,0 +1,13 @@
+==========================
+Django 2.2.3 release notes
+==========================
+
+*Release date TBA*
+
+Django 2.2.3 fixes several bugs in 2.2.2.
+
+Bugfixes
+========
+
+* Fixed a regression in Django 2.2 that prevented using filter clauses with
+  certain aggregation functions (:ticket:`30542`).
```

Comment:
```
* Fixed a regression in Django 2.2 where :class:`~django.db.models.Avg`,
  :class:`~django.db.models.StdDev`, and :class:`~django.db.models.Variance`
  crash with ``filter`` argument (:ticket:`30542`).
```

I added stub release notes for 2.2.3, please rebase your PR.

Reply:
Thanks!